### PR TITLE
Mark root element with attr

### PR DESCRIPTION
**Pull request by vvscode**
https://github.com/oblador/loki/issues/109

https://github.com/oblador/loki/projects/1#card-5208258

With this update, I'm able to use  next code in styles 

```css
  *[loki-test] .loki-invisible {
    opacity: 0;
  }
```
or from js:
```javascript
if (window._isLokiTest) {
    // loki specific logic
}
```

### CODE REVIEW

File: src/targets/chrome/add-loki-session-marker.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const addLokiSessionMarker = window =>
+  window.document.querySelector(':root').setAttribute('loki-tests', true);
```
Comment (oblador) [inline src/targets/chrome/add-loki-session-marker.js]:
```suggestion
  window.document.querySelector(':root').setAttribute('loki-test', true);
```

Somehow feels this should be in singularis.

**Comment by oblador** [comment]
Should we maybe expose a function that one could call to test this procedurally? Like `isLokiTest()`. 

Also should we support this on native perhaps?

File: src/targets/chrome/add-loki-session-marker.js (hunk 1 above)
Comment (vvscode) [reply src/targets/chrome/add-loki-session-marker.js]: updated

**Comment by vvscode** [comment]
@oblador I have no experience in using Loki with native apps. As far as I know ( from my experience in RN) - there it's easier to implement with `isLokiTest` function. However, I have a vision of implementing it for web, not for native 

Are you going share contributors guide?

**Comment by oblador** [comment]
If you're OK with doing a `isLokiTest()` implementation for web I can do one for native in a separate PR.

**Comment by vvscode** [comment]
@oblador  see no point for function. just have `window._isLokiTest` field is enough for me

**Comment by vvscode** [comment]
@oblador what do you think?

**Comment by vvscode** [comment]
Yep. But importing the module in tests make impossible import that module out of tests. Just because loki-tests is something aside from the main application. I understand your point of view, but don't understand how it's possible to implement

**Comment by oblador** [comment]
Hmm, I wouldn't want to litter my components with test code, but rather use it in the story itself to pass a prop. Regardless, importing a function or checking a global would be just as bad to do in your production code? If you want to be able to remove it with a minifier etc, then I guess some thing like `process.env.LOKI_TEST` would be better no?

**Comment by vvscode** [comment]
but it also pollution of global variable, isn't it? And more than that - you have to care about passing that prop to the components

**Comment by vvscode** [comment]
and in web it should be `window.preocess.env.LOKI_TEST`?

**Comment by oblador** [comment]
`process.env.LOKI_TEST` is pollution yes, but at least there's tooling to have it replaced with the primitive value and thus tree-shakeable. 

Passing it as a prop in stories is to me fine because stories is your test code and that's where it belongs IMHO. You could also have a context provider for all your stories with some generic theme settings.

**Pull request merged**